bindgen/bindgen.bzl

```diff
@@ -26,6 +26,14 @@ def rust_bindgen_library(
     """Generates a rust source file for `header`, and builds a rust_library.
 
     Arguments are the same as `rust_bindgen`, and `kwargs` are passed directly to rust_library.
+
```

Comment:
Why add/paste docs here instead of in the doc attrs that go to generated docs?

Reply:
Are you referring specifically to `Arguments are the same as `rust_bindgen`, and `kwargs` are passed directly to rust_library.` or this entire docstring?

Reply:
entire docstring

Reply:
This does go into generated docs though: https://bazelbuild.github.io/rules_rust/rust_bindgen.html#rust_bindgen_library

Reply:
That doc doesn't include the attr docs? I haven't been following how this works now, but the redundancy seems bad since it's extremely easy for them to fall out of sync

Reply:
The feature I'm hoping for is [bazelbuild/stardoc#27](https://github.com/bazelbuild/stardoc/issues/27) which should unlock the ability to refer to documentation from other rules without having to duplicate it. I don't think it's acceptable to have public rules undocumented (plus, it makes the rendered docs look bad). I also don't think the interface changes so frequently that there's a high risk of the docs getting outdated. I think this is very easily something that can be handled in pull requests. Your thoughts?

Reply:
it's fine, would just prefer to avoid the duplication if we could

Reply:
Yeah, I agree. In this case I feel the docs are worth having. In the future this could become more streamlined though.